src/Symfony/Component/Console/Input/InputArgument.php

```diff
@@ -38,7 +39,7 @@ class InputArgument
      *
      * @throws InvalidArgumentException When argument mode is not valid
      */
-    public function __construct(string $name, int $mode = null, string $description = '', string|bool|int|float|array $default = null)
+    public function __construct(string $name, int $mode = null, string $description = '', string|bool|int|float|array $default = null, \Closure|iterable|null $values = null)
```

Comment:
`|null` should be removed per our CS policy
I'd rather use array than iterable here

Reply:
Updated to `array|\Closure`